game-app/game-core/src/main/java/games/strategy/triplea/ui/EditPanel.java

```diff
@@ -528,12 +539,16 @@ public void actionPerformed(final ActionEvent event) {
             }
             int newTotal = oldTotal;
             try {
-              newTotal = Integer.parseInt(pusField.getText());
+              newTotal = Integer.parseInt(totalField.getText());
             } catch (final Exception e) {
               // ignore malformed input
             }
+            final IEditDelegate delegate = EditPanel.this.frame.getEditDelegate();
+            // Use the old changePUs() API since changeResource() was only added in 2.6.
```

Comment:
I think you can assume 2.6 given the other considerations of how we want players to upgrade to 2.6

Reply:
OK